helmclient.go

```diff
@@ -137,7 +137,7 @@ func (c *Client) DeleteRelease(ctx context.Context, releaseName string, options
 
 		return nil
 	}
-	b := backoff.NewExponential(backoff.ShortMaxWait, backoff.ShortMaxInterval)
+	b := backoff.NewExponential(30*time.Second, 5*time.Second)
```

Comment:
Doesn't it make sense to call max tries instead? There is no really much room for exponential backoffs.

Reply:
WDYT as below? ```
b := backoff.NewMaxRetries(5, 10*time.Second)```

Reply:
I'd rather say `(10, 5*time.Second)`. 5 seconds is the max with your previous proposal. Have you tested that is enough?

Reply:
I will try to do that in `ginger` after an agreement.

Reply:
We want the backoff to be short so other CRs can be processed. But not so short that the operator can't do what it needs to do. Pawel's proposal lgtm but it needs to be tested.

I would test this with the Ingress Controller chart as it is complex. So check it can be created, updated and deleted with these settings.